### PR TITLE
removes bay teargas

### PR DESCRIPTION
**Pull request by CaraS212**
Apparently teargas was extra strong due to both bays gas and ours in tgitems.dm being applied at the same time in 1 case. at least thats what i understand nyx to have said. This comments out bay's spicy gas and uses our mild recipe.

### CODE REVIEW

File: code/game/objects/items/weapons/grenades/chem_grenade.dm
Comment (scrdest) [inline code/game/objects/items/weapons/grenades/chem_grenade.dm:1]: Probably would have been better to change the recipe here and comment out the pure-Urist version.

For one, this means we don't have to worry as much about maintaining the object itself. For another, we should be using Initialize(), not New(), where possible.

**Review by scrdest — CHANGES_REQUESTED** [review]

**Comment by Glloyd** [comment]
how intense is the bay version compared to our old one when it's not being double applied? like scr says, it makes more sense to just get rid of our old one, but i don't know if it's even necessary to change the bay values

**Comment by CaraS212** [comment]
glloyd: bays is 2x as strong as ours

File: code/game/objects/items/weapons/grenades/chem_grenade.dm
Comment (CaraS212) [reply code/game/objects/items/weapons/grenades/chem_grenade.dm:1]: copy, ill change it such that the bay file uses an amount that is halved so it is equal to our old stuff. should i comment out ours in the tgitems file?

**Comment by CaraS212** [comment]
see description of most recent commit

File: code/game/objects/items/weapons/grenades/chem_grenade.dm [hunk 1]
```diff
@@ -278,13 +278,12 @@
...
-	var/obj/item/reagent_containers/glass/beaker/large/B1 = new(src)
-	var/obj/item/reagent_containers/glass/beaker/large/B2 = new(src)
-	B1.reagents.add_reagent(/datum/reagent/phosphorus, 40)
-	B1.reagents.add_reagent(/datum/reagent/potassium, 40)
-	B1.reagents.add_reagent(/datum/reagent/capsaicin/condensed, 40)
-	B2.reagents.add_reagent(/datum/reagent/sugar, 40)
-	B2.reagents.add_reagent(/datum/reagent/capsaicin/condensed, 80)
+	var/obj/item/reagent_containers/glass/beaker/B1 = new(src)
+	var/obj/item/reagent_containers/glass/beaker/B2 = new(src)
+	B1.reagents.add_reagent(/datum/reagent/capsaicin, 30)
```
Comment (Glloyd) [inline code/game/objects/items/weapons/grenades/chem_grenade.dm:283]: what is the difference mechanically between capsaicin and capsaicin/condensed?

Comment (CaraS212) [reply code/game/objects/items/weapons/grenades/chem_grenade.dm:283]: condensed is what is supposed to be in a grenade and teargas, normal is what is in food and just gives taste and pain if you eat too much of it.

**Comment by CaraS212** [comment]
i double checked and pepperspray does use condensed capsaicin, i believe prior to the merge the grenades did as well, knowing from taking them apart. 
also its been a week on something minor, i have no idea what i was doing code wise beyond making teargas not be able to gas out an entire deck.

**Pull request merged**